core/trino-main/src/test/java/io/trino/sql/planner/assertions/WindowFunctionMatcher.java

```diff
@@ -70,6 +71,7 @@ private boolean windowFunctionMatches(Function windowFunction, WindowFunction ex
     {
         return expectedCall.name().equals(windowFunction.getResolvedFunction().signature().getName().getFunctionName()) &&
                 WindowFrameMatcher.matches(expectedCall.frame(), windowFunction.getFrame(), aliases) &&
+                Objects.equals(expectedCall.orderBy(), windowFunction.getOrderingScheme()) &&
```

Comment:
why it's called orderBy in one place and orderingScheme in other?

Reply:
IDK :) was like that in AggregationFunctionMatcher. Will make it consistent.